esdb/proposed_event.go

```diff
@@ -0,0 +1,21 @@
+package esdb
+
+import (
+	uuid "github.com/gofrs/uuid"
+)
+
+type ContentType int
+
+const (
+	BinaryContentType ContentType = 0
+	JsonContentType   ContentType = 1
+)
+
+// ProposedEvent ...
+type ProposedEvent struct {
```

Comment:
@YoEight, any particular reason why we do not call this type `EventData` as we do in other clients?